server.js

```diff
@@ -28,20 +48,23 @@ const fastify = Fastify({
   logger: true,
 });
 
+Sentry.setupFastifyErrorHandler(fastify);
 await fastify.register(envPlugin);
 
-fastify.register(
-  helmet,
-  {
-    contentSecurityPolicy: {
-      directives: {
-        "connect-src": ["'self'", "sdk.openui5.org"],
-        "script-src": isLocalDev ? ["'self'", "'unsafe-inline'"] : ["'self'"],
-        "frame-ancestors": [fastify.config.FRAME_ANCESTORS]
-      },
-    }
-  }
-)
+let sentryHost = null;
```

Comment:
```suggestion
let sentryHost = '';
```
otherwise it will appear as `connect-src 'self' sdk.openui5.org null` in the CSP rules

Reply:
Did you test it? For me it is then just not part of it.